l10n_ch_base_bank/tests/test_bank.py

```diff
@@ -158,46 +174,38 @@ def test_set_ccp(self):
             'acc_number': None,
             'ccp': '10-8060-7',
         })
-        bank_acc.onchange_ccp_set_empty_acc_number()
+        bank_acc.onchange_ccp_set_acc_number()
 
         self.assertEqual(bank_acc.acc_number, '10-8060-7')
         self.assertEqual(bank_acc.bank_id, self.post_bank)
 
     def test_set_ccp_bank(self):
-        bank_acc = self.env['res.partner.bank'].new({
-            'partner_id': self.partner.id,
-            'acc_number': None,
-            'bank_id': self.bank.id,
-            'ccp': '10-8060-7'
-        })
-        bank_acc.onchange_ccp_set_empty_acc_number()
+        # we crate bank account
+        with self.env.do_in_onchange():
+
+            bank_acc = self.env['res.partner.bank'].new({
+                'partner_id': self.partner.id,
+                'acc_number': None,
+                'bank_id': self.bank.id,
+                'ccp': '10-8060-7'
+            })
+            bank_acc.onchange_ccp_set_acc_number()
+            # in result we should get new ccp number as we have bank_id and
+            # this he has ccp, new acc_number
 
-        self.assertEqual(bank_acc.acc_number, 'Bank/CCP Camptocamp')
-        self.assertEqual(bank_acc.bank_id, self.bank)
+            self.assertEqual(bank_acc.acc_number, 'Camptocamp/CCP 10-8060-7')
+            self.assertEqual(bank_acc.ccp, '10-8060-7')
+            self.assertEqual(bank_acc.bank_id, self.bank)
 
     def test_set_ccp_post(self):
```

Comment:
```suggestion
    def test_create_ccp_post(self):
```

We might still want both the create and the onchange tests.